admin/src/app/home/orders/stock-orders/receive/services/indexdb.ts

```diff
@@ -0,0 +1,33 @@
+import Dexie from "dexie";
+
+// Declare Database
+class ProductDatabase extends Dexie {
```

Comment:
Why do we have two database classes  ?
Please use I class/database to contain both tables

Reply:
Both table have different functions and that's why it'll be confusing to use a single class